content/docs/rfcs/17/README.md

```diff
@@ -161,9 +166,10 @@ message RateLimitProof {
 
 message WakuMessage {
   bytes payload = 1;
-  string contentTopic = 2;
-  uint32 version = 3;
-  double timestamp = 4;
+  string content_topic = 2;
```

Comment:
Note: The WakuMessage proto now refers to the proto in 14/WAKU2-MESSAGE

Reply:
Is it an fyi? can you please give more context? :)

Reply:
Yup, it's an fyi. Looks like 14/WAKU2-MESSAGE's protobuf definition changed, so I just updated it here